spec/routes/web/auth_spec.rb

```diff
@@ -76,6 +76,18 @@
 
     visit reset_link
     expect(page.title).to eq("Ubicloud - Reset Password")
+
+    fill_in "Password", with: "#{TEST_USER_PASSWORD}_new"
+    fill_in "Password Confirmation", with: "#{TEST_USER_PASSWORD}_new"
+
+    click_button "Reset Password"
+
+    expect(page.title).to eq("Ubicloud - Login")
+
+    fill_in "Email Address", with: TEST_USER_EMAIL
+    fill_in "Password", with: "#{TEST_USER_PASSWORD}_new"
+
+    click_button "Sign in"
```

Comment:
hmm... can we do remove_all_active_sessions_except_current_one(), so that we would be signed off. Not a big deal, just a bit annoying.

Reply:
Done